app/webpacker/components/Live/Admin/DoubleCheck/index.jsx

```diff
@@ -0,0 +1,151 @@
+import AttemptResultField from '../../../EditResult/WCALive/AttemptResultField/AttemptResultField';
+import updateRoundResults from '../../api/updateRoundResults';
+import getRoundResults from '../../api/getRoundResults';
+import Loading from '../../../Requests/Loading';
+
+export default function Wrapper({
+  competitionId, competitors, round,
+}) {
+  return (
+    <WCAQueryClientProvider>
+      <Competitors
+        eventId={round.event.id}
+        competitionId={competitionId}
+        competitors={competitors}
+        round={round}
+      />
+    </WCAQueryClientProvider>
+  );
+}
+
+function zeroedArrayOfSize(size) {
+  return Array(size).fill(0);
+}
+
+function Competitors({
+  competitionId, competitors, round, eventId,
+}) {
+  const event = events.byId[eventId];
+  const solveCount = event.recommendedFormat().expectedSolveCount;
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [registrationId, setRegistrationId] = useState(competitors[0].id);
+  const [attempts, setAttempts] = useState(zeroedArrayOfSize(solveCount));
+
+  const { data: results, isLoading } = useQuery({
+    queryKey: [round.id, 'results'],
+    queryFn: () => getRoundResults(round.id, competitionId),
+  });
+
+  const {
+    mutate: mutateUpdate, isPending: isPendingUpdate,
+  } = useMutation({
+    mutationFn: updateRoundResults,
+  });
+
+  const handleSubmit = async () => {
+    mutateUpdate({
+      roundId: round.id, registrationId, competitionId, attempts,
+    });
+  };
+
+  const handleAttemptChange = (index, value) => {
+    const newAttempts = [...attempts];
+    newAttempts[index] = value;
+    setAttempts(newAttempts);
+  };
+
+  const handleRegistrationIdChange = useCallback((_, { value }) => {
+    setRegistrationId(value);
+    const alreadyEnteredResults = results.find((r) => r.registration_id === value);
+    setAttempts(alreadyEnteredResults.attempts.map((a) => a.result));
+  }, [results]);
+
+  const currentCompetitor = useMemo(() => {
+    if (results) {
+      return competitors.find((r) => r.id === results[currentIndex].registration_id);
+    }
+    return {};
+  }, [competitors, currentIndex, results]);
+
+  useEffect(() => {
+    if (results) {
+      setAttempts(results[currentIndex].attempts.map((a) => a.result));
+    }
+  }, [currentIndex, results]);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  return (
+    <Grid>
+      <Grid.Column width={1} verticalAlign="middle">
+        { currentIndex !== 0 && <Button onClick={() => setCurrentIndex((oldIndex) => oldIndex - 1)}>{'<'}</Button>}
+      </Grid.Column>
+      <Grid.Column width={7}>
+        <Segment loading={isPendingUpdate}>
+          <Form>
+            <Form.Select
+              label="Competitor"
+              placeholder="Competitor"
+              value={currentCompetitor.id}
+              search={(inputs, value) => inputs.filter((d) => d.text.toLowerCase().includes(value.toLowerCase()) || parseInt(value, 10) === d.registrationId)}
```

Comment:
I'm surprised that SemUI doesn't handle lowercasing natively. I am 97% sure I've added myself as a dummy organizer to comps before by using `gregor billing`. Does the `deburr` option change anything?